examples/ex1b_generate_thermal_sims_monoblock.py

```diff
@@ -0,0 +1,83 @@
+'''
+================================================================================
+Example: Generate Perturbed 3D Monoblock Temperature Field Simulations using MOOSEHERDER and Stratified Sobol Sampling of parameter space.
+
+icarus:  Intelligent Compatibility Analyser for Reactor Uncertain Simulations
```

Comment:
Nice backronym 😆

Though I would prefer to avoid implying `icarus` is only suitable for reactor simulations, (and fusion comms don't like us using the word reactor anyway). Personally I don't think the name needs to be an acronym for anything.